staff/asix/full-task-app-1.0.0/src/index.js

```diff
@@ -0,0 +1,38 @@
+require('dotenv').config()
+
+// const axios = require('axios')
+const express = require ('express')
+const app = express()
+const axiosCall = require ('./axios.js')
+
+const bodyParser = require ('body-parser')
+const formBodyParser = bodyParser.urlencoded({ extended: false })
+
+app.set('view engine', 'pug')
+
+app.get('/', (req, res) => {
+
+    axiosCall.list(res)
+})
+
+app.post('/add', formBodyParser, (req, res) => {
+    const {text} = req.body
+
+    axiosCall.create(res, text)
+})
+
+app.get('/done/:id', (req, res) => {
+    const { id } = req.params
+
+    axiosCall.edit(res, id)
```

Comment:
Probably axiosCall.edit is not so semantic,  I would try to use updateDoneTask instead

Reply:
Totally agreed with Robert.